searx/templates/simple/preferences/language.html

```diff
@@ -13,9 +13,9 @@
       {% for sxng_tag,lang_name,country_name,english_name,flag in sxng_locales | sort(attribute=1) -%}
         <option value="{{ sxng_tag }}"
                 {%- if sxng_tag == current_language %} selected="selected" {%- endif -%}>
-                {%- if flag -%}{{ flag }} {% endif -%}
                 {{ lang_name }}{%- if country_name -%}-{{ country_name }}{%- endif -%}
-                {{- ' ' -}}[{{sxng_tag}}]{{- '' -}}
+                {{- ' ' -}}[{{sxng_tag}}]{{- ' ' -}}
```

Comment:
Currently if the flag doesn't exist, an extra space will be added at the end. I can fix this, but also I doubt it will really affect UI/UX at all.